code/game/turfs/open.dm

```diff
@@ -353,6 +353,7 @@
 
 /turf/open/rad_act(pulse_strength)
 	if (air.gases[/datum/gas/carbon_dioxide] && air.gases[/datum/gas/oxygen])
+		pulse_strength = min(pulse_strength,air.gases[/datum/gas/carbon_dioxide][MOLES]*1000,air.gases[/datum/gas/oxygen][MOLES]*2000)
```

Comment:
could you leave a comment on these and similar limits elsewhere in the PR indicating that this is ensuring that matter is conserved appropriately?